common/djangoapps/student/helpers.py

```diff
@@ -270,6 +262,31 @@ def get_next_url_for_login_page(request):
     return redirect_to
 
 
+def get_redirect_to(request):
+    """
+    Determine the redirect url and return if safe
+    :argument
+        request: request object
+
+    :returns: redirect url if safe else None
+    """
+    static_file_format = '\w+\.(jpeg|jpg|gif|png|ico)'
```

Comment:
@rlucioni Here is what @maxrothman wrote on the ticket:
`Allowing assets to be pointed to by "next" allows 3rd party sites to get information about a user on edx.org. They can put an <img> tag on their site that points to edx.org/login?next=some_asset.jpg. If the user is logged in, the image will load, and if not, the login page will be served and the image will fail to load. By detecting the success of the image load, 3rd party sites can determine whether a user is logged in on edx.org.`

Do you agree with the above? If so, do you think it's a security problem we should address?